core/src/main/resources/openapi-specs/eth2/signing/schemas.yaml

```diff
@@ -415,6 +417,8 @@ components:
       properties:
         version:
           type: string
+          enum:
+            - ALTAIR
           description: 'version to identify block request type.'
           example: ALTAIR
```

Comment:
nit: do we need these examples now that's it's using an enum?

Reply:
not really ... it would simply add an example in rendered openapi spec. It has no effect on the validation.

Reply:
we can safely remove it.